core/block/editor/state/state.go

```diff
@@ -686,6 +686,8 @@ func (s *State) apply(fast, one, withLayouts bool) (msgs []simple.EventMessage,
 	}
 
 	msgs = s.processTrailingDuplicatedEvents(msgs)
+
+	sortEventMessages(msgs)
```

Comment:
It seems we should insert sorting before trailing dups analysis. Or we don't need this analysis at all? :grinning: